File: Crio/Array/15_Conversion/Conversion.cpp
#include<bits/stdc++.h>
using namespace std;

int countConversionBits(long long a, long long b) {
	// CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
	long long xorr = a ^ b;
	int result = __builtin_popcountll(xorr);
	return result;
	// CRIO_SOLUTION_END_MODULE_L1_PROBLEMS
}

int main() {
	long long a , b;
	cin >> a >> b;
	int answer = countConversionBits(a, b);
	cout << answer;
	return 0;
}

File: Crio/Array/15_Conversion/Conversion.py

def countConversionBits(a, b):
# CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
    xorr = a^b
    result = bin(xorr).count('1')
    return result
# CRIO_SOLUTION_END_MODULE_L1_PROBLEMS

def main():
    a, b = map(int, input().split())
    answer = countConversionBits(a, b)
    print(answer)

if __name__ == "__main__":
    main()
    
